torch/quantization/fx/quantization_patterns.py

```diff
@@ -337,15 +348,31 @@ def convert(
             else:
                 assert (
                     len(self.conv_node.args) == 7
-                ), "only conv2d calls with all arguments specified is support right now in debug=False option"
+                ), "only conv2d or conv3d calls with all arguments specified is support right now in debug=False option"
                 args = load_arg(quantized=[0, 1])(self.conv_node.args)
                 # pack weight
                 weight = load_arg(quantized=True)(self.conv_node.args[1])
                 other_args = load_arg(quantized=False)(self.conv_node.args[2:])
                 prepack_args = tuple([weight] + list(other_args))
+
+                assert self.node_target in [
+                    torch.nn.functional.conv2d,
+                    torch.nn.functional.conv3d,
+                ], "Only conv2d and conv3d are supported"
+                self.prepack_corresponding_module = (
+                    torch.ops.quantized.conv3d_prepack
+                    if torch.nn.functional.conv3d == self.node_target
+                    else torch.ops.quantized.conv2d_prepack
+                )
+                self.quantized_corresponding_module = (
+                    torch.ops.quantized.conv3d
+                    if torch.nn.functional.conv3d == self.node_target
+                    else torch.ops.quantized.conv2d
+                )
```

Comment:
nit: can you just use local variables for these?

Reply:
Done